tileserver/cache.py

```diff
@@ -148,7 +149,11 @@ def release_lock(self, cache_key):
 
     def set(self, cache_key, data):
         key = self._generate_key('data', cache_key)
-        self.client.set(key, data)
```

Comment:
Instead of using a pipeline, just pass the expiration into the `set()` call as the arg after `data`. To be explicit you could use `ex=self.expires`. [See client source](https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L1138).